databuilder/databuilder/task/neo4j_staleness_removal_task.py

```diff
@@ -64,6 +70,32 @@ class Neo4jStalenessRemovalTask(Task):
 
     """
 
+    delete_stale_nodes_statement = textwrap.dedent("""
+        MATCH (target:{{type}})
+        {staleness_condition}{{extra_condition}}
```

Comment:
an we make staleness_condition one of `{{` as well so it has to be named when formatting. Just makes things more uniform IMO. Same points about the other statements.

Reply:
Agreed about the uniformity but the reason I put the single brace around `staleness_condition` (and the reason it was single braces before it was named) is bc each time `.format` is called on the statement it removes one set of curly braces, so since we have the `_decorate_staleness` function that handles populating the staleness condition first, and then after that the target `type` and `extra_condition` are left with a single `{}` and are populated in each of the other functions after. Do you think this is alright, or an alternative would be to bring all the formatting into the `_decorate_staleness` function if that is preferable?